<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeterReader</class>
 <widget class="QMainWindow" name="MeterReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>475</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #435BC2;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>FIRE DETECTION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>QUANTIZATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>411</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>The following are user guidelines:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>1.To select a video, press the select video button and the selected video will load above the button.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>680</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>4.To make the detections press the run detections button.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>2.To select a camera feed, press the select camera feed button and the selected feed will load above the select video button.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>3.To add fire respodent and stakeholder emails use the respodents buttons. The add respondents appends new respodents to the old list while create new respodents overrides, deletes the old respodents and adds a new list with new respodents.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>475</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 250, 238);
background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>465</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Selected video</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sphoto">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>401</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid #EBF0FC;
border-radius: 15px;
color: rgb(235, 240, 252);</string>
     </property>
     <property name="text">
      <string>No videofeed uploaded</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 25px;
background:#435BC2;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}</string>
     </property>
     <property name="text">
      <string>Select video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="algorithm">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>680</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 25px;
background:#fff;
color: #000;
border: 3px solid #EBF0FC;

}
QPushButton:hover{
background-color:rgb(0,0,0);
color: #FFF;
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>Run Detections</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>450</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 25px;
background:#435BC2;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}</string>
     </property>
     <property name="text">
      <string>Select camera feed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>600</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 25px;
background:#435BC2;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}</string>
     </property>
     <property name="text">
      <string>Add respodents</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid #EBF0FC;
border-radius: 15px;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>600</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 25px;
background:#435BC2;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}</string>
     </property>
     <property name="text">
      <string>create new respodents</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
